sdk/openai/azure-ai-openai/src/samples/java/com/azure/ai/openai/ReadmeSamples.java

```diff
@@ -56,6 +56,24 @@ public void createAsyncClientKeyCredential() {
         // END: readme-sample-createAsyncClientKeyCredential
     }
 
+    public void createNonAzureSyncClientWithApiKey() {
+        // BEGIN: readme-sample-createNonAzureOpenAISyncClientApiKey
+        OpenAIClient client = new OpenAIClientBuilder()
+            .credential(new NonAzureOpenAIKeyCredential("{key}"))
+            .endpoint("{endpoint}")
+            .buildClient();
+        // END: readme-sample-createNonAzureOpenAISyncClientApiKey
+    }
+
+    public void createNonAzureAsyncClientWithApiKey() {
+        // BEGIN: readme-sample-createNonAzureOpenAIAsyncClientApiKey
+        OpenAIAsyncClient client = new OpenAIClientBuilder()
+            .credential(new NonAzureOpenAIKeyCredential("{key}"))
+            .endpoint("{endpoint}")
```

Comment:
Why is endpoint needed for public OpenAI service?

Reply:
My bad. It should be removed.